contest/b_nom_de_groupe/sol/b_nom_de_groupe.cpp

```diff
@@ -0,0 +1,49 @@
+// Solution by Louis Sugy
+
+#include <iostream>
+#include <string>
+
+int score(std::string name) {
+  int l = name.size();
+  bool bonus = false;
+  const char *name_array = name.c_str();
```

Comment:
Why are you using `const char*` instead of `std::string`'s `operator[]`?

Reply:
It was slightly faster in some cases, I'm not sure why. Nothing can beat using a raw pointer 😏 
But i'll revert for the sake of simplicity, the difference is not significant nor consistent.